app/assets/stylesheets/components/_theme.scss

```diff
@@ -93,9 +170,7 @@ li{
 }
 
 html{
-  font-family: sans-serif;
-  font-size: Theme-fontSize(base);
```

Comment:
Even with this approach of yours, you should enforce the font-size on the html element